src/patternfly/components/List/list.scss

```diff
@@ -43,4 +83,16 @@
       }
     }
   }
+
+  &.pf-m-bordered {
+    > * {
+      padding-bottom: var(--pf-c-list--m-bordered--li--PaddingBottom);
+      border-bottom: var(--pf-c-list--m-bordered--li--BorderBottomWidth) solid var(--pf-c-list--m-bordered--li--BorderBottomColor);
+    }
+
+    > :last-child {
+      padding-bottom: 0;
+      border-bottom: none;
```

Comment:
FYI a shorter way to write this would be `border: 0;` or `border-bottom: 0;`, they're both valid.